planning_domains_api.py

```diff
@@ -104,6 +104,33 @@ def simple_query(qs):
     else:
         return res['result']
 
+def create_col(name, description, domain_set, ipc):
+    """Create a new collection in the database"""
+    attribute = ''  # Unknown for now
+    value = ''      # Same Unknown for now
+
+    params = {'user': USER_EMAIL,
+              'password': USER_TOKEN,
+              'ipc': ipc,
+              'desc': description,
+              'domain_set': domain_set
+    }
+    path = "/{}/create_collection/".format(name)
```

Comment:
POST: api.planning.domains/json/<formalism>/collections

Reply:
![Screenshot 2023-03-30 215307](https://user-images.githubusercontent.com/71029388/229002888-026ea520-aaf7-493f-84d4-cee1f0f57d0b.png)
This is what I have so far, any changes needed?